### PR TITLE
Adding Next and Previous Buttons

### LINKED ISSUE #973: Add Next and Previous buttons to exposure pages
From @slamassa:
Maybe this functionality exists and I missed it, but is it possible to navigate to the “next” image when you have one image selected? I didn’t find a way to do so, so I use the “back” button on my browser to get back to the list of images, and then because I have a goldfish memory, I don’t remember which image in an observation I was inspecting and which the next one is. With ~50 images in 1 observation, it’s really easy for me to lose track of what I inspected and what I need to inspect. (Not sure if this rises to the level of “critical” but I figured I’d bring it up.)

### PR DESCRIPTION
**Pull request by mfixstsci**
Using `django.core.paginator`

### CODE REVIEW

File: jwql/website/apps/jwql/views.py [hunk 1]
```diff
@@ -37,12 +37,14 @@
...
 """
 
+from collections import defaultdict
 import csv
 import os
 
 from bokeh.layouts import layout
 from bokeh.embed import components
 from django.http import HttpResponse, JsonResponse
+from django.core.paginator import Paginator
```
Comment (BradleySappington) [inline jwql/website/apps/jwql/views.py]: Paginator not used, can be deleted

File: jwql/website/apps/jwql/views.py [hunk 2]
```diff
@@ -960,9 +965,24 @@ def view_image(request, inst, file_root, rewrite=False):
...
+    rootnames = get_rootnames_for_instrument_proposal(inst, prop_id)
+    file_root_list = defaultdict(list)
+
+    for root in rootnames:
+        try:
+            file_root_list[(filename_parser(root)['observation'])].append(root)
+        except KeyError:
+            pass
+
+    file_root_list = {key : sorted(file_root_list[key]) for key in sorted(file_root_list)}
```
Comment (BradleySappington) [inline jwql/website/apps/jwql/views.py]: currently an initial sort.  Sorting to follow page sorting will be implemented with a future PR?

**Review by BradleySappington — APPROVED** [review]
Looks good @mfixstsci

File: jwql/website/apps/jwql/views.py (hunk 2 above)
Comment (mfixstsci) [reply jwql/website/apps/jwql/views.py]: So this was my first attempt at sorting and just figured I would do it alphabetically by default. Should we go ahead and remove it and implement it in another pr?

File: jwql/website/apps/jwql/views.py (hunk 1 above)
Comment (mfixstsci) [reply jwql/website/apps/jwql/views.py]: Thanks -- I'll make sure to do this.

File: jwql/website/apps/jwql/views.py (hunk 2 above)
Comment (BradleySappington) [reply jwql/website/apps/jwql/views.py]: Leaving it this way is a-ok.  Some sort of known sort is better than leaving it out.  Verifying that we weren't pursuing advanced sorting in this PR.

**Comment by mfixstsci** [comment]
Looks like I merged the TA monitor, accidentally force pushed, and then re-merged it again.

**Pull request merged**